test/spec_rewindable_input.rb

```diff
@@ -81,38 +88,43 @@
     }.should.not.raise
   end
 
+  after do
```

Comment:
Couldn't we use `teardown` instead of `after`? Since the `before` blocks are changed to `def setup` further down?